src/workerd/api/node/process.c++

```diff
@@ -137,4 +139,56 @@ void ProcessModule::exitImpl(jsg::Lock& js, int code) {
   js.logWarning(kj::str(err.get(js, "stack"_kj)));
 }
 
+kj::String ProcessModule::getCwd(jsg::Lock& js) {
+  auto cwd = getCurrentWorkingDirectory();
+  if (cwd == nullptr) {
+    return kj::str("/");
+  }
+  return kj::str("/", cwd.toString());
```

Comment:
`cwd` here is a `kj::PathPtr` correct? If so, I believe you can just `return cwd.toString(true)` here